### PR TITLE
Docs: Installation

### PR DESCRIPTION
**Pull request by scohen**
Added installation documentation for vscode and emacs

### CODE REVIEW

**Comment by scohen** [comment]
Partially fixes #228

File: pages/installation.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Lexical, and is not supported at this time.
+
+
+## Visual Studio Code
+
+Click on the extensions button on the sidebar, then search for
+`lexical`, then click `install`.  To change to a local executable, go
+to `Settings -> Extensions -> Lexical` and type
+`/my/home/projects/lexical/_build/dev/rel/lexical` into the text box
+in the `Server: Release path override` section.
```
Comment (Blond11516) [inline pages/installation.md:91]:
```suggestion
## Visual Studio Code

Click on the extensions button on the sidebar, then search for
`lexical`, then click `install`.  By default, the extension will automatically
download the latest version of Lexical.

To change to a local executable, go to `Settings -> Extensions -> Lexical` and
type `/my/home/projects/lexical/_build/dev/rel/lexical` into the text box in
the `Server: Release path override` section.
```
I think this makes it clearer that the user doesn't have to do anything beyond installing the extension by default.

File: pages/installation.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+(use-package elixir-mode
+  :ensure t
+  :custom
+  (lsp-elixir-server-command '("/my/home/projects/_build/dev/rel/lexical/start_lexical.sh")))
+
+```
+
+Restart emacs, and Lexical should start when you open a file with an
+`.ex` extension.
```
Comment (Blond11516) [inline pages/installation.md:76]:
```suggestion
Restart emacs, and Lexical should start when you open a file with a
`.ex` extension.
```
very nitpicky: Not a native english speaker, but I read this as "a file with a dot-e-x extension" so "a" reads better to me.

**Review by Blond11516 — APPROVED** [review]
Don't know emacs, no idea if the instructions are correct but the rest looks good.

**Pull request merged**